cli/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ _Released 12/19/2023 (PENDING)_
 
 **Bugfixes:**
 
+- Fixed asset capture for Test Replay for requests that are routed through service workers and for requests that span tests when test isolation is off. This addresses an issue where styles would not being applied properly in Test Replay or `cy.intercept` would not work properly. Fixes [#28516](https://github.com/cypress-io/cypress/issues/28516) and [#28545](https://github.com/cypress-io/cypress/issues/28545).
```

Comment:
```suggestion
- Fixed asset capture for Test Replay for requests that are routed through service workers and for requests that span tests when test isolation is off. This addresses an issue where styles were not being applied properly in Test Replay or `cy.intercept` would not work properly. Fixes [#28516](https://github.com/cypress-io/cypress/issues/28516) and [#28545](https://github.com/cypress-io/cypress/issues/28545).
```

Not sure if this is a complete correction of the second sentence. Please cross-check!

Reply:
Thanks @MikeMcC399. Updated.